testing/suite/integration.go

```diff
@@ -225,10 +225,20 @@ func setup(t *testing.T) error {
 		t.Errorf("unexpected error: %v\n%s\n%s", err, createClusterCmd.GetOutput(), createClusterCmd.GetError())
 		return err
 	}
+	// create pod
+	createPodCmd := NewKubectlCommandLine("apply", "-f", "testdata/prereq/pod-withlabel.yaml")
+	err = createPodCmd.Exec()
+	if err != nil {
+		t.Errorf("unexpected error: %v\n%s\n%s", err, createPodCmd.GetOutput(), createPodCmd.GetError())
+		return err
+	}
 	return nil
 }
 
 func tearDown(t *testing.T) {
+	// delete pod
+	NewKubectlCommandLine("delete", "-f", "test-create-git-annotations-workload-pod", "-n", TestingNamespace).Exec()
```

Comment:
`kubectl delete --all pods -n TestingNamespace` (cleanup all resources, not individual ones)

Reply:
Done